### PR TITLE
docs: mentioned the etcd gRPC gateway

### PR DESCRIPTION
**Pull request by tokers**
### What this PR does / why we need it:

etcd v3 only supports gRPC as the messaing protocol, only enabling gRPC gateway, APISIX can communicate to it with Restful APIs.

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

### CODE REVIEW

File: doc/install-dependencies.md [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 Note
 ====
-- Since v2.0 Apache APISIX would not support the v2 protocol storage to etcd anymore, and the minimum etcd version supported is v3.4.0.
+- Since v2.0 Apache APISIX would not support the v2 protocol storage to etcd anymore, and the minimum etcd version supported is v3.4.0. What's more, etcd v3 uses gRPC as the messaging protocol, while Apache APISIX uses HTTP(s) to communicate with etcd cluster, so be sure the [etcd gRPC gateway](https://etcd.io/docs/v3.4.0/dev-guide/api_grpc_gateway/) is enabled.
```
Comment (spacewander) [inline doc/install-dependencies.md:31]: `HTTP(s)`, `s` should be uppercase.

Comment (tokers) [reply doc/install-dependencies.md:31]: OK

**Comment by nic-chen** [comment]
please update Chinese version together, thanks.

**Review by juzhiyuan — COMMENTED** [review]
LGTM and please update the Chinese doc.

**Comment by tokers** [comment]
@nic-chen @juzhiyuan Updated the Chinese doc also, please check it out :)

**Comment by nic-chen** [comment]
LGTM

**Review by nic-chen — APPROVED** [review]

File: doc/zh-cn/install-dependencies.md [hunk 2]
```diff
@@ -27,7 +27,7 @@
 
 注意
 ====
-- Apache APISIX 从 v2.0 开始不再支持 `v2` 版本的 etcd，并且 etcd 最低支持版本为 v3.4.0。
+- Apache APISIX 从 v2.0 开始不再支持 `v2` 版本的 etcd，并且 etcd 最低支持版本为 v3.4.0。更要重要的是，因为 etcd v3 使用 gPRC 作为消息传递协议，而 Apache APISIX 使用 HTTP(S) 与 etcd 集群交互，因此请确保 [etcd gRPC gateway](https://etcd.io/docs/v3.4.0/dev-guide/api_grpc_gateway/) 功能是被启用的。
```
Comment (juzhiyuan) [inline doc/zh-cn/install-dependencies.md:30]: 1. 因此请使用 etcd 3.4.0+。
2. Typo: 更重要的是
3. Use 通信 instead of 交互
4. 请确保启用 xxx。

**Review by juzhiyuan — CHANGES_REQUESTED** [review]

File: doc/zh-cn/install-dependencies.md (hunk 2 above)
Comment (tokers) [reply doc/zh-cn/install-dependencies.md]: Fixed.

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**